storage_backblaze.go

```diff
@@ -0,0 +1,193 @@
+/*
+ * knoxite
+ *     Copyright (c) 2016, Christian Muehlhaeuser <muesli@gmail.com>
+ *     Copyright (c) 2016, Nicolas Martin <penguwingithub@gmail.com>
+ *
+ *   For license see LICENSE.txt
+ */
+
+package knoxite
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"gopkg.in/kothar/go-backblaze.v0"
+)
+
+// StorageBackblaze stores data on a remote Backblaze
+type StorageBackblaze struct {
+	url            url.URL
+	chunkFile      string
+	snapshotFile   string
+	repositoryFile string
+	// bucketPrefix   string
+	bucket    *backblaze.Bucket
+	backblaze *backblaze.B2
+}
+
+// NewStorageBackblaze returns a
+func NewStorageBackblaze(URL url.URL) (*StorageBackblaze, error) {
+	// Checking username and password
+	if URL.User.Username() == "" {
+		return &StorageBackblaze{}, nil
```

Comment:
Shouldn't this return an error instead of just an empty StorageBackblaze instance?